packages/apollo-engine-reporting/src/durationHistogram.ts

```diff
@@ -48,8 +60,9 @@ export class DurationHistogram {
     }
   }
 
-  constructor(buckets?: number[]) {
-    this.buckets = Array<number>(DurationHistogram.BUCKET_COUNT).fill(0);
+  constructor(initSize = 74, buckets?: number[]) {
+    let arrayInitSize = Math.max(buckets?.length || 0, initSize);
```

Comment:
use const, not let

`?.` in TS, a brave new world! How exciting.

Does this constructor let you pass buckets but no initSize?  Maybe the parameters should be in the other order?

Reply:
It doesn't I felt like we were more likely to want to pass in the `initSize` than buckets so I thought this would work better. I don't see us passing `buckets` that often while doing reporting?

Reply:
Maybe this should just be an options object instead of multiple optional parameters.